x/virtualgroup/keeper/msg_server.go

```diff
@@ -56,6 +56,10 @@ func (k msgServer) CreateGlobalVirtualGroup(goCtx context.Context, req *types.Ms
 		return nil, sptypes.ErrStorageProviderNotFound.Wrapf("The address must be operator address of sp.")
 	}
 
+	if !sp.IsInService() {
```

Comment:
should also allow Maintenance SP otherwise it is not able to conduct self test if no GVG created yet?

Reply:
Fixed. Does sp in maintenance can send swapout tx?

Reply:
it is not allowed.

Reply:
yes, make sense. Please review this pr again. thx.